### PR TITLE
Fix dependencies

### PR DESCRIPTION
**Pull request by chewu**
**Overall change:**

Update the following dependencies:

"@bbc/psammead-calendars": "^2.0.14",
"@bbc/psammead-grid": "^1.1.15",
"@bbc/psammead-locales": "^4.1.9",
"@bbc/psammead-radio-schedule": "^3.0.6",
"@bbc/psammead-social-embed": "^1.1.3",
"@bbc/psammead-timestamp-container": "^4.0.1",
"helmet": "^3.23.1",
"react-lazyload": "^2.6.8",
"@testing-library/dom": "^7.16.2",
"@testing-library/jest-dom": "^5.9.0",
"@testing-library/jest-dom": "^5.10.1",
"@testing-library/react": "^10.2.1",
"@testing-library/react": "^10.3.0",
"chalk": "^4.1.0",
"cypress": "4.8.0",
"eslint-config-airbnb": "^18.2.0",
"eslint-plugin-import": "^2.21.2",
"eslint-plugin-jsx-a11y": "6.2.3",
"eslint-plugin-prettier": "^3.1.4",
"lint-staged": "^10.2.11",
"stylelint": "^13.6.1",
"terser-webpack-plugin": "^3.0.6",
"webpack-cli": "^3.3.12"

**Code changes:**

Update the dependency version numbers in package.json and package-lock.json

---

- [x] I have assigned myself to this PR and the corresponding issues
- [x] I have added labels to this PR for the relevant pod(s) affected by these changes
- [x] I have assigned this PR to the Simorgh project

**Testing:**

- [ ] Automated (jest and/or cypress) tests added (for new features) or updated (for existing features)
- [ ] If necessary, I have run the local E2E non-smoke tests relevant to my changes (`CYPRESS_APP_ENV=local CYPRESS_SMOKE=false npm run test:e2e:interactive`)
- [ ] This PR requires manual testing

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -165,25 +165,25 @@
...
+    "eslint-config-airbnb": "^18.2.0",
     "eslint-config-prettier": "^6.11.0",
     "eslint-import-resolver-alias": "^1.1.2",
     "eslint-plugin-cypress": "^2.11.1",
-    "eslint-plugin-import": "^2.21.1",
+    "eslint-plugin-import": "^2.21.2",
     "eslint-plugin-json": "^2.1.1",
-    "eslint-plugin-jsx-a11y": "^6.2.3",
-    "eslint-plugin-prettier": "^3.1.3",
+    "eslint-plugin-jsx-a11y": "6.2.3",
```
Comment (jamesdonoh) [inline package.json:185]: Accidental removal of `^`?

Comment (chewu) [reply package.json:185]: I locked this to that particalur version as it was breaking on of the tests, however with the latest version the tests now pass.

**Comment by amywalkerdev** [comment]
There's a couple more that have come up in this weeks dependabot sweep that probably want adding too: https://github.com/bbc/simorgh/pulls?q=is%3Apr+is%3Aopen+label%3Adependencies

I wouldn't do the Mocha one bundled in this though as its a major version.

I have added these as well except Mocha.

**Comment by chewu** [comment]
@joshcoventry @amywalker1987 @jamesdonoh 
Note: Pinned eslint-plugin-jsx-a11y@6.2.3 as it was failing on the licence check for:
```
Problems with the licenses for these dependencies:
  language-subtag-registry@0.3.20
    License:     ODC-By-1.0
    Repository:  https://github.com/mattcg/language-subtag-registry
    Publisher:   undefined
    Url:         undefined
```

Should this be added to the licence excpetion list?

**Comment by jamesdonoh** [comment]
@chewu I'm not comfortable adding an exception for that package without us getting legal advice, if the older version doesn't have any licence issues then let's pin back. Please could you create an issue to capture the task that we need to go and get this licence reviewed/approved before we can upgrade `eslint-plugin-jsx-a11y`?

@jamesdonoh Will do.

File: package.json [hunk 2]
```diff
@@ -85,20 +85,20 @@
     "@bbc/psammead-live-label": "^1.0.0",
     "@bbc/psammead-locales": "^4.1.8",
     "@bbc/psammead-media-indicator": "^4.0.7",
-    "@bbc/psammead-media-player": "^2.7.11",
-    "@bbc/psammead-most-read": "^4.1.3",
+    "@bbc/psammead-media-player": "^2.7.10",
```
Comment (jamesdonoh) [inline package.json]: Are we deliberately downgrading this?

Comment (chewu) [reply package.json]: @jamesdonoh Oh. Will fix. Thanks.

**Comment by jamesdonoh** [comment]
I'm also a bit concerned by the large number of changes in this PR where a lazyload `div` is now nested inside a `span`, e.g. https://github.com/bbc/simorgh/pull/6953/files#diff-10d522e660e63776bac9b1c19e989c2aL1613-R1635. Yesterday we were discussing DOM size/nesting warnings we are getting in Lighthouse reports and this will only exacerbate that problem.

Do we know what is cauing this change, is it something pulled in from `psammead` or the bump to `react-lazyload`? And as it's a front-end change, has it been a11y tested?

**Pull request closed without merging**